### PR TITLE
fix: Position of info icon in calculating card

### PR DESCRIPTION
**Pull request by memoyil**
Before:

Screenshot 2021-05-14 at 22 52 50

After: 

Screenshot 2021-05-14 at 22 52 22

### CODE REVIEW

File: src/views/Predictions/components/RoundCard/CalculatingCard.tsx [hunk 1]
```diff
@@ -36,11 +36,9 @@ const CalculatingCard: React.FC<CalculatingCardProps> = ({ round }) => {
...
             <Flex alignItems="center" justifyContent="center" flexDirection="column">
               <Spinner size={96} />
-              <Flex mt="8px">
-                <span ref={targetRef}>
-                  <TooltipText>{t('Calculating')}</TooltipText>
-                  <InfoIcon />
-                </span>
+              <Flex mt="8px" ref={targetRef}>
+                <TooltipText>{t('Calculating')}</TooltipText>
+                <InfoIcon />
```
Comment (hachiojidev) [inline src/views/Predictions/components/RoundCard/CalculatingCard.tsx:41]: Can you add a margin left of 4px on the Info Icon

**Review by hachiojidev — CHANGES_REQUESTED** [review]
Thanks, been meaning to get around to this. Just one thing.

File: src/views/Predictions/components/RoundCard/CalculatingCard.tsx (hunk 1 above)
Comment (memoyil) [reply src/views/Predictions/components/RoundCard/CalculatingCard.tsx]: Done

File: src/views/Predictions/components/RoundCard/CalculatingCard.tsx [hunk 2]
```diff
@@ -36,11 +36,9 @@ const CalculatingCard: React.FC<CalculatingCardProps> = ({ round }) => {
...
             <Flex alignItems="center" justifyContent="center" flexDirection="column">
               <Spinner size={96} />
-              <Flex mt="8px">
-                <span ref={targetRef}>
-                  <TooltipText>{t('Calculating')}</TooltipText>
-                  <InfoIcon />
-                </span>
+              <Flex mt="8px" ref={targetRef}>
+                <TooltipText>{t('Calculating')}</TooltipText>
+                <InfoIcon ml="5px" />
```
Comment (hachiojidev) [inline src/views/Predictions/components/RoundCard/CalculatingCard.tsx:41]: Our spacing should be in units of 4 pixels.

**Review by hachiojidev — CHANGES_REQUESTED** [review]

File: src/views/Predictions/components/RoundCard/CalculatingCard.tsx (hunk 2 above)
Comment (memoyil) [reply src/views/Predictions/components/RoundCard/CalculatingCard.tsx]: Sorry :D my bad. Fixed now

**Review by hachiojidev — APPROVED** [review]

**Pull request merged**